.github/workflows/main.yml

```diff
@@ -3,8 +3,6 @@ name: main
 on:
   push:
     branches:
```

Comment:
i presume we need the branches key?

Reply:
I think so, because there's a `tags` sibling key too. But irrelevant now because we're allowing `"test-me-*"` instead :)